app/views/shared/_footer.html.erb

```diff
@@ -3,6 +3,9 @@
     <div class="copyright">
       Copyright 2014–<%= Date.today.year %> The Regents of the University of California, All Rights Reserved.
     </div>
+    <div class="version">
+      Version <span title="<%= GIT_SHA %>"><%= BRANCH %> updated <%= LAST_DEPLOYED %></span>
```

Comment:
I don't mind having the SHA in parentheses after the version string.  Or maybe I'm just worried that using `title` like this isn't properly semantic.

Reply:
I think it might look too weird to have the SHA down there. Do you mind if we just get rid of it?